src_c/transform.c

```diff
@@ -812,9 +808,8 @@ surf_flip(PyObject *self, PyObject *args, PyObject *kwargs)
 
     if (!xaxis) {
         if (!yaxis) {
-            for (loopy = 0; loopy < surf->h; ++loopy)
-                memcpy(dstpix + loopy * dstpitch, srcpix + loopy * srcpitch,
-                       surf->w * surf->format->BytesPerPixel);
+            assert(srcpitch == dstpitch);
```

Comment:
This isn't always true though. srcpitch can != dstpitch

Reply:
I'm not sure how they would be different. The two surfaces have the same format, and presumably SDL would allocate pitches the same way on two same sized surfaces with the same format each time.

But in the interest of safety, and to not prod with things that don't need prodding: #3416